packages/shopware-6-client/src/services/customerService.ts

```diff
@@ -238,6 +240,30 @@ export async function updatePassword(
   await apiService.post(getCustomerUpdatePasswordEndpoint(), params);
 }
 
+/**
+ * @alpha
+ */
+export interface CustomerResetPasswordParam {
+  email: string;
+  storefrontUrl?: string;
+}
+
+/**
+ * Reset a customer's password
+ *
+ * @throws ClientApiError
+ * @alpha
+ */
+export async function resetPassword(
+  params: CustomerResetPasswordParam
+): Promise<void> {
+  if (!params.storefrontUrl) {
```

Comment:
1. I have added optional chaining here but it makes a problem with branch coverage due to 
https://github.com/istanbuljs/istanbuljs/issues/516

Just added a workaround with `/* istanbul ignore next */` FYI :)

2. 
> This will require to add unit test invoking method with no params like
> `const` result = await resetPassword(null as any);

I do not get this. `null as any` will cause TypeError due to inconsistency with `CustomerResetPasswordParam` or I just do not understand something? :D

3. After successful operation, modal just closes and email is sent. I was modeling on login operation.

Reply:
Here's what I meant with this test, turned out it has to be another if statement. What we needed to test:
1. invocation with no parameter provided
2. invocation without any parameters provided

please take a look at this commit: https://github.com/DivanteLtd/shopware-pwa/pull/756/commits/29bc3289da88c6514988885652dd219028a56150

Why should we test for 2? Because library compiles to JS and someone might unintentionally ignore param for method. We need to be resistant to this kind of behaviour:)